bindings/jupyroot/python/JupyROOT/helpers/utils.py

```diff
@@ -498,7 +527,11 @@ def _canJsDisplay(self):
 
     def _getJsCode(self):
         # produce JSON for the canvas
-        json = produceCanvasJson(self.drawableObject)
+        json = ""
```

Comment:
Note that you don't need this initialization line, since `json` is both assigned in the `if` and `else` cases, it will for sure be assigned when the conditional clause ends.

Reply:
I was not sure about scope rules in python, therefore just to be on safe side

Reply:
`if`s do not create a new scope in Python, so we are good with no initialization here.